src/service-worker.js

```diff
@@ -1,8 +1,11 @@
 'use strict';
 
-/* eslint-env serviceworker */
+/* eslint-env browser, serviceworker */
+
+importScripts('./scripts/analytics.js');
 
 self.addEventListener('push', function(event) {
+  self.analytics.trackEvent('push-received');
```

Comment:
All values are filtered out before being passed to analytics in the payload. The eventAction is the only required field.